code/__DEFINES/mobs.dm

```diff
@@ -954,6 +954,9 @@ GLOBAL_LIST_INIT(layers_to_offset, list(
 /// Types of bullets that mining mobs take full damage from
 #define MINING_MOB_PROJECTILE_VULNERABILITY list(BRUTE)
 
+/// Helper macro that determines if the mob is at the threshold to start vomitting due to high toxin levels
+#define AT_TOXIN_VOMIT_THRESHOLD(mob) (mob.getToxLoss() > 45 && mob.nutrition > 20)
```

Comment:
Should we set `45` and `20` here to named constants (e.g. `VOMIT_TOXIN_THRESHOLD` and `REQUIRED_NUTRITION_TO_VOMIT`) in case we need it in the future/make it easier to tweak? it seems like they were chosen pretty arbitrarily and it'd be nice to have a nice tweak, as well as just make it so people can hook into the values for features down the line.

Reply:
If someone wants to change the threshold they can change it in this very macro, it would be the same thing as making them defines

Reply:
yeah i was just wondering about accessing each comparison component outside of this scope but i think it's a can we can kick down the road